doc/tech/glossary.md

```diff
@@ -47,41 +47,41 @@ To help guide the reader, this glossary provides an introduction to the terminol
   - **Virtual Storage resource**: virtualised non-volatile storage allocated to a virtualised computation environment hosting a **VNFC**
   - **Virtual Networking resource**: routes information among the network interfaces of a virtual compute resource and physical network interfaces, providing the necessary connectivity
 - **Hypervisor**: A piece of software with management components that allows the user to partition the underlying physical resources and allocate them to Virtual Machines. Typically, the hypervisor is managed by a cloud management software such as OpenStack.
-- **NFVI Software Profile (NFVI SW Profile)**: defines the behaviour, capabilities and metrics provided by an NFVI Software Layer
-- **NFVI Software Configuration (NFVI SW Configuration)**: a set of settings (Key:Value) that are applied/mapped to **NFVI** SW deployment.
+- **Cloud Infrastructure Software Profile**: defines the behaviour, capabilities and metrics provided by a cloud infrastructure Software Layer
```

Comment:
Should we change NFVI (Software/Hardware) Profile to Cloud Infrastructure (Software/Hardware) Profile accross whole CNTT?

Reply:
Yes, I know @tomkivlin is preparing a PR to go through those terms and neutralising them as on the back of #1168

Reply:
okay